environment.yml

```diff
@@ -20,7 +20,7 @@ dependencies:
 - jinja2~=3.0
 - jsonschema~=3.0
 - kiwipy[rmq]~=0.7.7
-- importlib-metadata~=4.13
+- importlib-metadata~=6.0
```

Comment:
@sphuber I've bumped this all the way to 6.x and test still pass. WDYT?

Reply:
It should be compatible with Python 3.8 and up and not have any other significant dependency restrictions, so I think this is fine: https://github.com/python/importlib_metadata/blob/main/setup.cfg